libpod/container.go

```diff
@@ -586,9 +587,36 @@ func (c *Container) Kill(signal uint) error {
 }
 
 // Exec starts a new process inside the container
-// Returns fully qualified URL of streaming server for executed process
-func (c *Container) Exec(cmd []string, tty bool, stdin bool) (string, error) {
-	return "", ErrNotImplemented
+func (c *Container) Exec(tty, privileged bool, env, cmd []string, user string) error {
+	var capList []string
+	c.lock.Lock()
+	defer c.lock.Unlock()
+
+	if err := c.syncContainer(); err != nil {
+		return err
+	}
+
+	conState := c.state.State
+
+	if conState != ContainerStateRunning {
+		return errors.Errorf("cannot attach to container that is not running")
+	}
+	if privileged {
```

Comment:
Ok I went and investigated Docker and it looks like the only security effected by --privileged is Capabilities. Which I think is wrong, since on docker run --privileged also effects SELinux, AppArmor, SECCOMP.  But effecting those would need to be wired into runc.  

I guess this is the best we can do for now.